app/components/Puzzle.tsx

```diff
@@ -296,6 +297,14 @@ export const Puzzle = ({
   useEventListener('blur', prodPause);
 
   const [muted, setMuted] = usePersistedBoolean('muted', false);
+  const [colorPref, setColorPref] = useDarkModeControl();
+
+  const toggleColorPref = useCallback(
+    () => {
+      setColorPref(colorPref === 'light' ? 'dark' : 'light');
```

Comment:
I think a user may have to tap this 2 times if:
- the theme is currently set to the browser/OS default
- the browser/OS default is dark

I'm not sure if there's a good workaround for that that minimizes the # of options on the menu. Open to suggestions!